handlers/load_test_handler.go

```diff
@@ -118,7 +126,7 @@ func (h *Handler) LoadTestUsingSMPHandler(w http.ResponseWriter, req *http.Reque
 	}
 	loadTestOptions.AllowInitialErrors = true
 
-	h.loadTestHelperHandler(w, req, testName, "istio", testUUID, prefObj, loadTestOptions, provider)
+	h.loadTestHelperHandler(w, req, profileID, testName, "istio", "", prefObj, loadTestOptions, provider)
```

Comment:
instead of `"istio"`, I think this should be read from the incoming profile's body.

Reply:
Good catch will add that here.

Reply:
Not sure how do I do that. There's no field for Service Mesh Name in `SMP.PerformanceTestConfig` struct
// @navendu-pottekkat @leecalcote